src/elements/helper/GridHelper.ts

```diff
@@ -10,6 +11,10 @@ export function CalculateGridInformation(designItem: IDesignItem) {
   const computedStyle = getComputedStyle(designItem.element);
   const rows = computedStyle.gridTemplateRows.split(' ');
   const columns = computedStyle.gridTemplateColumns.split(' ');
+  const rowUnits = [];
+  (<HTMLElement>designItem.element).style.gridTemplateRows.split(' ').forEach(rowWidth => rowUnits.push(getCssUnit(rowWidth)));
+  const columnUnits = [];
+  (<HTMLElement>designItem.element).style.gridTemplateColumns.split(' ').forEach(columnHeight => columnUnits.push(getCssUnit(columnHeight)));

```

Comment:
da hab ich das erstmal so gemacht, auch wenns glaube ich nicht ganz richtig ist, weil ich ja alles vom computed style nehmen sollte, aber die einheiten muss ich ja irgendwo her bekommen und vom computed style kommt ja alles nur in pixel